modules/test/playwright/tests/saml-web/main/saml.spec.ts

```diff
@@ -3565,4 +3559,18 @@ test('LPD-37323 AC2/AC4 TC2: User switches between apps. When already logged in
 	// Verify user is redirected back to restricted resource
 
 	expect(await spInstancePage.url()).toContain(spNewPageUrl);
+
+	await systemSettingsPage.goToSystemSetting('Login', 'Login');
```

Comment:
This logic needs to be in the `afterAll/Each` method, otherwise if this test fails, the `enable prompt` setting will remain system-wide.  That, or you can adjust it on all sites required.